README.md

```diff
@@ -13,58 +13,37 @@ Kafka De/Serializer using avro and Confluent's Schema Registry
 
 ```clojure
 (ns kafka-avro-confluent.readme-test
-  (:require [clojure.walk :as walk]
-            [kafka-avro-confluent.deserializers :as des]
-            [kafka-avro-confluent.schema-registry-client :as reg]
-            [kafka-avro-confluent.serializers :as ser])
-  (:import kafka_avro_confluent.Serde))
+  (:require [kafka-avro-confluent.v2.deserializer :as des]
+            [kafka-avro-confluent.v2.serializer :as ser]))
 
 ;; initialise the Confluent Schema Registry client:
-(def schema-registry
-  (reg/->schema-registry-client
-   {;; NOTE auth optional!
-    ;; :username "mr.anderson"
-    ;; :password "42"
-    :base-url "http://localhost:8081"}))
+(def config
+  {;; NOTE auth optional!
+   ;; :schema-registry/username "mr.anderson"
+   ;; :schema-registry/password "42"
+   :schema-registry/base-url "http://localhost:8081"})
 
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 ;; # Deserializer
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 ;; implements org.apache.kafka.common.serialization.Deserializer
 
-(des/->avro-deserializer schema-registry)
+(des/->avro-deserializer config)
 
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 ;; # Serializer
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 ;; implements org.apache.kafka.common.serialization.Serializer
-(def some-value-schema {:type "record" :name "Foo"})
 
 ;; meant to be used as a value-serializer in a KafkaProducer
-(ser/->avro-serializer schema-registry some-value-schema)
```

Comment:
ok so this is the motivation: it now takes only the config and not the schema at instantiation